content/blog/self_host_local_ai_assistant_with_jan_and_pinggy.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Self-host Local AI Assistant with Jan and Pinggy"
+description: "Learn how to run Jan, a ChatGPT alternative, locally and share it online using Pinggy. A complete guide to setting up your private AI assistant with full data control and privacy."
+date: 2025-06-19T14:00:00+05:30
+draft: false
+tags: ["Jan", "Pinggy", "Self-Hosted AI", "Local AI", "AI Assistant", "Privacy", "Jan.ai"]
+og_image: "images/self_host_local_ai_assistant_with_jan_and_pinggy/jan_banner.webp"
+schemahowto: "PHNjcmlwdCB0eXBlPSJhcHBsaWNhdGlvbi9sZCtqc29uIj4KewogICJAY29udGV4dCI6ICJodHRwczovL3NjaGVtYS5vcmciLAogICJAdHlwZSI6ICJIb3dUbyIsCiAgIm5hbWUiOiAiU2VsZi1ob3N0IExvY2FsIEFJIEFzc2lzdGFudCB3aXRoIEphbiBhbmQgUGluZ2d5IiwKICAiZGVzY3JpcHRpb24iOiAiTGVhcm4gaG93IHRvIHJ1biBKYW4sIGEgQ2hhdEdQVCBhbHRlcm5hdGl2ZSwgbG9jYWxseSBhbmQgc2hhcmUgaXQgb25saW5lIHVzaW5nIFBpbmdneS4iLAogICJpbWFnZSI6ICJodHRwczovL3BpbmdneS5pby9pbWFnZXMvc2VsZl9ob3N0X2xvY2FsX2FpX2Fzc2lzdGFudF93aXRoX2phbl9hbmRfcGluZ2d5L2phbl9iYW5uZXIud2VicCIsCiAgInN0ZXAiOiBbCiAgICB7CiAgICAgICJAdHlwZSI6ICJIb3dUb1N0ZXAiLAogICAgICAidGV4dCI6ICJEb3dubG9hZCBhbmQgaW5zdGFsbCBKYW4gZnJvbSBqYW4uYWkiCiAgICB9LAogICAgewogICAgICAiQHR5cGUiOiAiSG93VG9TdGVwIiwKICAgICAgInRleHQiOiAiRG93bmxvYWQgYW4gQUkgbW9kZWwgZnJvbSB0aGUgSHViVGFiIgogICAgfSwKICAgIHsKICAgICAgIkB0eXBlIjogIkhvd1RvU3RlcCIsCiAgICAgICJ0ZXh0IjogIlN0YXJ0IGNoYXR0aW5nIGluIHRoZSBOZXcgQ2hhdCB0YWIiCiAgICB9LAogICAgewogICAgICAiQHR5cGUiOiAiSG93VG9TdGVwIiwKICAgICAgInRleHQiOiAiU2hhcmUgb25saW5lIHdpdGggUGluZ2d5IHVzaW5nOiBzc2ggLXAgNDQzIC1SMDpsb2NhbGhvc3Q6MTMzNyAtdCBxckBhLnBpbmdneS5pbyB1Okhvc3Q6bG9jYWxob3N0OjEzMzciCiAgICB9CiAgXQp9Cjwvc2NyaXB0Pg=="
+outputs:
+  - HTML
+  - AMP
+---
+{{< image "self_host_local_ai_assistant_with_jan_and_pinggy/jan_banner.webp" "Self-host Local AI Assistant with Jan and Pinggy" >}}
```

Comment:
change the image. instead of icon crop the inner window from here: https://jan.ai/assets/images/homepage/app-frame-light-fixed.png